pkg/virt-controller/network/hotplug.go

```diff
@@ -83,3 +99,25 @@ func ClearDetachedInterfaces(specIfaces []v1.Interface, specNets []v1.Network, s
 
 	return ifaces, vmispec.FilterNetworksByInterfaces(specNets, ifaces)
 }
+
+func isPersistentIPNetwork(networkClient k8scnicncfiov1.K8sCniCncfIoV1Interface, namespace string, network v1.Network) (bool, error) {
+	if !vmispec.IsSecondaryMultusNetwork(network) {
+		return false, nil
+	}
+
+	nads, err := GetNetworkAttachmentDefinitionByName(networkClient, namespace, []v1.Network{network})
+	if err != nil {
+		return false, fmt.Errorf("failed retrieving network attachment definitions: %w", err)
+	}
+
+	conf, err := libipam.GetPersistentIPsConf(nads[network.Name])
+	if err != nil {
+		return false, fmt.Errorf("failed retrieving netConf: %w", err)
+	}
+
+	if conf.AllowPersistentIPs {
+		return true, nil
+	}
+
+	return false, nil
+}
```

Comment:
can't we move this logic to the new pkgs and just invoke from here the right func / method ?

Reply:
ref https://github.com/kubevirt/kubevirt/pull/11410#discussion_r1606537778